frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostDetails.tsx

```diff
@@ -140,7 +140,7 @@ const BareMetalHostDetails: React.FC<BareMetalHostDetailsProps> = ({
           <dl>
             <dt>Status</dt>
             <dd>
-              <BareMetalHostStatus {...status} />
+              <BareMetalHostStatus {...status} maintenance={nodeMaintenance} host={host} />
```

Comment:
Is this change necessary? The `{...status}` already includes  maintenance or host. See `host-status.ts`